src/settings/toml/manifest.rs

```diff
@@ -336,8 +340,12 @@ impl Manifest {
             name: self.name.clone(), // Inherited
             kv_namespaces: get_namespaces(self.kv_namespaces.clone(), preview)?, // Not inherited
             durable_objects: self.durable_objects.clone(), // Not inherited
-            site: self.site.clone(), // Inherited
-            vars: self.vars.clone(), // Not inherited
+            migrations: self // Top level
+                .migrations
+                .clone()
+                .map(|migrations| Migrations { migrations }),
```

Comment:
Wait...is this gonna work if migrations is `None`? I assume it will but I don't see any like, filtering

Reply:
yep, map will pass `None` through